Neptune.Database/dbo/Views/dbo.vWaterQualityManagementPlanAnnualReport.sql

```diff
@@ -0,0 +1,66 @@
+create view dbo.vWaterQualityManagementPlanAnnualReport
+as
+
+select 
+	   concat(wqmp.WaterQualityManagementPlanID,'_', wqmpv.WaterQualityManagementPlanVerifyID, '_', fv.FieldVisitID) as PrimaryKey,

```

Comment:
I don't think we need a PrimaryKey column anymore; that was an artifact of the old generator